src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -1104,6 +1104,12 @@ public void ArgumentParsing()
             Assert.True(parsedArgs.DisplayHelp);
             Assert.False(parsedArgs.SourceFiles.Any());
 
+            parsedArgs = CSharpCommandLineParser.ScriptRunner.Parse(new[] { "/version" }, _baseDirectory, s_defaultSdkDirectory);
```

Comment:
>parsedArgs [](start = 12, length = 10)

We should have a test that verifies the actual output of the compiler, including scenarios when other switches are present.